tests/test_surfaces.py

```diff
@@ -405,6 +405,7 @@ class TestZernikeRZToroidalSection:
     """Tests for ZernikeRZToroidalSection class."""
 
     @pytest.mark.unit
+    @pytest.mark.xfail(reason="GitHub issue 1127.")
```

Comment:
why is this failing now? Shouldn't a cross section be able to compute area?

Reply:
I was pushing this off to the PR that resolved issue 1127, but you are right that I should address this here.